frontend/src/lib/components/CommandPalette/index.tsx

```diff
@@ -28,9 +28,9 @@ export function CommandPalette(): JSX.Element | null {
             squeakAudio?.play()
         } else if (event.key === 'Escape') {
             event.preventDefault()
-            // First of all, exit flow
-            if (activeFlow) activateFlow(null)
-            // Else just erase input
+            // Return to previous flow
+            if (activeFlow) backFlow()
+            // If no flw, erase input
```

Comment:
nitpick: typo

Note: I think these 3 comments are pretty pointless, the code is well-written in that regard and the naming conveys the same intent. :+1: